scripts/download-release.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/sh
+COMMAND_NAME=${1:-terraform}
+TARGETPLATFORM=${2:-"linux/amd64"}
+DEFAULT_VERSION=${3:-"1.6.2"}
+AVAILABLE_VERSIONS=${4:-"1.6.2"}
+case "${TARGETPLATFORM}" in
+  "linux/amd64") ARCH=amd64 ;;
+  "linux/arm64") ARCH=arm64 ;;
+  "linux/arm/v7") ARCH=arm ;;
+  *) echo "ERROR: 'TARGETPLATFORM' value unexpected: ${TARGETPLATFORM}"; exit 1 ;;
+esac
+for VERSION in ${AVAILABLE_VERSIONS}; do
+  case "${COMMAND_NAME}" in
+    "terraform")
+      DOWNLOAD_URL_FORMAT=$(printf 'https://releases.hashicorp.com/terraform/%s/%s_%s' "$VERSION" "$COMMAND_NAME" "$VERSION")
+      COMMAND_DIR=/usr/local/bin/tf
+      ;;
+    "tofu")
```

Comment:
can we call this opentofu instead?

Reply:
We cannot cause despite the release name of `opentofu`, the binary name is `tofu`